bin/init_bin

```diff
@@ -3,5 +3,6 @@
 # usage: init_bin
 
 export PATH=$PATH:`pwd`
+export MANPATH=`pwd`/../man/:${MANPATH}
 echo "Successfully added `pwd` to PATH."
```

Comment:
I think we should also let the user know that we modified the `MANPATH` environment variable. Something along the lines of "Successfully added PipeViz man path to MANPATH"